tests/testsuite/registry_auth.rs

```diff
@@ -85,7 +84,7 @@ fn simple() {
         .build();
 
     let p = make_project();
-    cargo(&p, "build").with_stderr(SUCCESS_OUTPUT).run();
+    cargo(&p, "build").with_stderr_data(SUCCESS_OUTPUT).run();
```

Comment:
Is it possible to make `SUCCESS_OUTPUT` snapshot aware?

Reply:
I feel that this comment contradict to <https://github.com/rust-lang/cargo/pull/14149#discussion_r1653730915>.

Reply:
> Is it possible to make `SUCCESS_OUTPUT` snapshot aware?

Is this referring to using its own `str![]` instead of the same `SUCCESS_OUTPUT`?

Reply:
Got it, thanks!